staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_type.go

```diff
@@ -467,8 +468,7 @@ func (c *$.type|privatePlural$) List(opts $.ListOptions|raw$) (result *$.resultT
 		return
 	}
 
-	if !$.stringsContains|raw$(err.Error(), "forbidden") ||
-		!$.stringsContains|raw$(err.Error(), "no relationship found between node") {
+	if !($.errorForbidden|raw$(err) && $.stringsContains|raw$(err.Error(), "no relationship found between node")) {
```

Comment:
It would help code reading to add a comment that node authorizer will issue an error like "no relationship found between node ...". At first I thought there was a typo as the sentence "no relationship found between node" is incorrect in grammar. ( I had thought it was meant to be "no relationship found between nodeS")

Reply:
The error message is defined in authorization plug in, used by both api server and kubelet. This update is just using part of the error message.